### DISCUSSION TITLE
New deployment Instructions using Divio - Docker

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#3007] [Feature Requests]
Hello, 

I just finished the deployment of Directus v9 using Divio, a cloud solution for docker, it is using AWS for database and storage which is good, and the free version offers just enough to start testing. 

I would like to write the instruction on how to do it so it can be published. I am not part of Divio, but I found it easier to use instead of Heroku and better because we do not have to create an AWS account and worry about billing, so win win :)

Let me know, happy to do it.

**Comment by rijkvanzanten** [comment]
Go ahead! @benhaynes what would be the best place under docs?

**Reply by benhaynes** [reply]
This is so exciting, thank you so much @juancarlosjr97 !! ❤️

We could add this as a guide within the docs, or as an article on the upcoming website. We'll need to clearly define where things go and why.

**Reply by juancarlosjr97** [reply]
Awesome, I will write it and then we can think where to add it :)

**Reply by juancarlosjr97** [reply]
Hey guys, I wrote some instructions using Divio and make sure everything is fine, did I deployment and the live version is working fine,  and a user has been created for testing and showing Directus working :) 

https://github.com/juancarlosjr97/directus-v9-divio-example

If you like it, what is remaining of the guide is the following:

* Adding custom domain - optional
* Sync Test and Live with changes, importing/exporting database and files probably capable doing "Import/Export Endpoints" feature of Directus v9 remaining to be completed
* Adding local development using Docker

Let me know what do you think @benhaynes and @rijkvanzanten 😄

**Reply by benhaynes** [reply]
Thank you so much, and sorry for the late reply... busy times around here.

This looks great to me (full disclosure, I haven't had time to test it 😉 ). Since it's specific to an external service/platform, I think this would be best suited for our new/upcoming website blog. I can share a private link to the staging site if you'd like to check it out. We'll be hooking up Directus to the site next week, and then we can give you access to move this article there.

Does that sound like a plan? Thanks again for writing this — really well organized! 😄

**Reply by juancarlosjr97** [reply]
Awesome @benhaynes , sounds like a cool plan to post it on the new/upcoming website blog, and yes please, I would love to see it!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
